docs/rose-suite-run-proposal/future-cli-conventions.md

```diff
@@ -1,14 +1,37 @@
-Placeholder - populate in new PR
 # `cylc flow` CLI proposal
-Refers to discussion in [cylc-flow #1030](https://github.com/cylc/cylc-flow/issues/1030).
 
+## Purpose
+Specify how the Cylc8 command to replace `rose-suite-run` will work.
+
+## References
+[cylc-flow #1030](https://github.com/cylc/cylc-flow/issues/1030).
 
 ## Discussion of possible names for new command
 
+Possibilities include:
+
+| Name | For | Against |
+| --- | --- | --- |
+|`cylc run`||Already in use for another command|
```

Comment:
This will command will replace `cylc run` so this is just keeping the name the same?

Reply:
I'm not sure that this is in fact the plan. I think we may keep the old cylc run for legacy reasons, but I'm going to forward this on to @matthewrmshin  becuase I don't have a convincing answer.

Reply:
I'll resolve this discussion, as the point will be obvious to any of us reading the document.